README.md

```diff
@@ -279,8 +279,9 @@ Each of these commands have their own help, referenced by `helm vault {enc,dec,c
 |`-v`, `--verbose`|Verbose output||`enc`, `dec`, `clean`, `view`, `edit`, `install`, `template`, `upgrade`, `lint`, `diff`|
 |`-s`, `--secret-file`|File containing secrets for input, rather than using stdin, must end in `.yaml.dec`||`enc`|
 |`-f`, `--file`|The specific YAML file to be deleted, without `.dec`||`clean`|
-|`-e`, `--editor`|Editor name|Windows: `notepad`, macOS/Linux: `vi`|`edit`|
+|`-ed`, `--editor`|Editor name|Windows: `notepad`, macOS/Linux: `vi`|`edit`|
```

Comment:
I think this change will necessitate a new major version as it is a breaking change.

I'm going to add a CHANGELOG at the same time since I see there isn't one yet.

Reply:
ah good point, I just threw that in without really thinking... we could technically leave it but I think it makes sense to have environment as the only `-e` since it is used across more commands 

> I'm going to add a CHANGELOG 

ty👌